quickwit-telemetry/src/sender.rs

```diff
@@ -280,7 +280,7 @@ impl TelemetrySender {
 
 /// Check to see if telemetry is enabled.
 pub fn is_telemetry_enabled() -> bool {
-    std::env::var_os(crate::DISABLE_TELEMETRY_ENV_KEY).is_none()
+    !cfg!(test) || std::env::var_os(crate::DISABLE_TELEMETRY_ENV_KEY).is_none()
```

Comment:
I think the construct `cfg(test)` is not valid during integration test.
Can you please add 
```rust
    cmd.env(
        quickwit_telemetry::DISABLE_TELEMETRY_ENV_KEY,
        "disable-for-tests",
    )
```
to `tests::helpers::make_command`

Reply:
we could also use debug_assertions
https://doc.rust-lang.org/reference/conditional-compilation.html#debug_assertions

Reply:
Ooops I think I already addressed this ticket by setting the env key in the test.